packages/mobile_app/lib/common/ambiance/ambiance_subject_item.dart

```diff
@@ -0,0 +1,128 @@
+import 'common.dart';
+
+class AmbiacneSubject extends StatelessWidget {
+  final Function onOptionsTap;
+  final UserEntity subject;
+  final double compatibility;
+  const AmbiacneSubject({
+    Key key,
+    @required this.onOptionsTap,
+    @required this.subject,
+    @required this.compatibility,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final sign = subject.model.birth.astroSign;
+    final birthDate = subject.model.birth.toDateTime;
+    final isNegative = (compatibility - 50.0) < 0.0;
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        /// {NAME} {ROLE}
+        Padding(
+          padding: const EdgeInsets.only(
+            top: 16.0,
+            left: 16.0,
+          ),
+          child: Row(
+            children: [
+              Text(
+                "${localeText.you.capitalize()}",
```

Comment:
Why not `localeText.you.capitalize()`? 
In this case its better to fast fail (crash) in case there is no translation, because we actually rely that `localeText.you` should **always** exist